### DISCUSSION TITLE
7月2日续期付费一年以后，没有收到邮件，也没有更新到期日期。

### DISCUSSION THREAD

**Discussion by bulagemou** [getlantern/lantern-forum-cn#681] [Q&A]
7月3号凌晨完成支付 支付宝扣钱了 没收到付款确认邮件，也没收到专业版验证码

**Comment by MaxMKMax** [comment]
@bulagemou 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by bulagemou** [reply]
> @bulagemou 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您! https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
> 
> *备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

![image](https://github.com/getlantern/lantern-forum-cn/assets/174626770/712d41ee-be85-4dc3-b3b5-2415c62d52c2)
无法访问这个链接

**Comment by MaxMKMax** [comment]
@bulagemou 您好. 请您提供我支付单据和账号邮箱. 我手动帮您提交问题.

**Comment by bulagemou** [comment]
更新新版软件后，问题已解决，谢谢支持。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月23日(星期二) 凌晨0:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 7月2日续期付费一年以后，没有收到邮件，也没有更新到期日期。 (Discussion #681)





 
@bulagemou 您好. 请您提供我支付单据和账号邮箱. 我手动帮您提交问题.

**Reply by blurice** [reply]
兄弟，自动更新还是自己在官网下的？官网的好像闪退吧？

**Comment by bulagemou** [comment]
看到论坛上有同样闪退问题，安装的Win7版本的不闪退




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月23日(星期二) 晚上9:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 7月2日续期付费一年以后，没有收到邮件，也没有更新到期日期。 (Discussion #681)





 
兄弟，自动更新还是自己在官网下的？官网的好像闪退吧？
